### PR TITLE
06-func.md: Replace "center" function name

### PR DESCRIPTION
**Pull request by neurosutras**
This issue https://github.com/swcarpentry/python-novice-inflammation/issues/459 recommended either replacing the function name "center", or providing a better explanation for what it did. Here I replace "center" with a potentially more intuitive "offset_mean". Since the function name and its description are repeated many times throughout the lesson, the relationship between its name and its purpose have many opportunities to be made clear and that understanding reinforced.

### CODE REVIEW

**Comment by maxim-belkin** [comment]
Hi, Aaron! Thanks for the PR!

I like the idea and agree that `offset_mean` is better than `centered`. Personally, I would prefer `offset_mean_value` and also change `desired` to `new_mean_value`.

@MikeTrizna, what are your thoughts on this?

Aaron, is this, incidentally, part of instructor checkout?

**Comment by MikeTrizna** [comment]
@maxim-belkin, I agree to `new_mean_value` (or `target_mean_value`, etc.) is a good replacement for desired, but I think that `offset_mean` is fine for the function name. `offset_mean_value` seems like it would be a good variable name to store the results of the `offset_mean` function (although I don't think we do that in this lesson at all).

File: _episodes/06-func.md [hunk 1]
```diff
@@ -315,11 +316,11 @@ but since we don't know what the values ought to be,
 it will be hard to tell if the result was correct.
 Instead,
 let's use NumPy to create a matrix of 0's
-and then center that around 3:
+and then offset it to have a mean value of 3:
```
Comment (maxim-belkin) [inline _episodes/06-func.md]: I'd suggest changing `offset` to `offset its values`

**Comment by gvwilson** [comment]
This is a good change - there are a few minor comments, but I think it can be merged as-is.

**Comment by maxim-belkin** [comment]
@neurosutras just checking in to see if you'll have time to make the proposed changes.
It looks like you've enabled "Allow changes from maintainers", so we can also push to your branch if you don't have time at the moment. Just let us know. 

Thank you!

**Comment by neurosutras** [comment]
Hi @maxim-belkin , you are more than welcome to push to this branch and oversee it's merging to master. Thank you.

**Comment by maxim-belkin** [comment]
OK, thanks @neurosutras! Will do!

**Pull request closed without merging**